arm/platform-implementations/home.hpp

```diff
@@ -0,0 +1,122 @@
+#include <libhal-rmd/mc_x.hpp>
+#include <libhal-util/serial.hpp>
+#include <libhal-util/steady_clock.hpp>
+#include <libhal/accelerometer.hpp>
+#include <math.h>
+#define M_PI 3.14156265
+
+using namespace std::chrono_literals;
+namespace sjsu::arm {
+inline hal::degrees atan2_d(float y, float x)
```

Comment:
```suggestion
inline hal::degrees atan2_to_degrees(float y, float x)
```

Took me a bit to understand what the title meant without reading the definition.